diff --git a/node_modules/osu-api-extended/dist/api/v1/index.js b/node_modules/osu-api-extended/dist/api/v1/index.js
index 1632fb7..720462f 100644
--- a/node_modules/osu-api-extended/dist/api/v1/index.js
+++ b/node_modules/osu-api-extended/dist/api/v1/index.js
@@ -44,8 +44,6 @@ exports.user = {
 };
 const match_1 = __importStar(require("./routes/match"));
 exports.match = match_1.default;
-const replay_1 = __importStar(require("./routes/replay"));
-exports.replay = replay_1.default;
 exports.desc = {
     beatmap_set: set_1.description,
     beatmap_diff: diff_1.description,
@@ -54,6 +52,5 @@ exports.desc = {
     user_details: details_1.description,
     user_scores_category: scores_category_1.description,
     match: match_1.description,
-    replay: replay_1.description,
 };
 //# sourceMappingURL=index.js.map
\ No newline at end of file
